### ISSUE TITLE
FireFTP opens unexpectedly

### ISSUE THREAD

**Issue by leskaye** [mimecuvalo/fireftp#192]
I maintain this web page which is a mash-up of various weather information
http://www.shobdongliding.co.uk/weathertoday.html
It includes some text data downloads from NOAA
For the last month when I navigate to the web page FireFTP opens unexpectedly and creates a temporary guest account on the NOAA server. 
My own web page has not been modified since last July and opens as expected in other web browsers. 
Please fix this if it is a bug or if it is a feature, how can I stop FireFTP opening when navigating to the page?

**Comment by Amosss** [comment]
Having the same problem with the reported URL and also with this one:
http://www.mako.co.il/news-world/international-q1_2017/Article-27113945d740a51004.htm

I see this post is 7 days old, I hope it will be handled soon.

**Comment by leskaye** [comment]
Example of the problem web page code which seems to be activating FireFTP
<iframe src="ftp://tgftp.nws.noaa.gov/data/forecasts/taf/stations/EGBB.TXT"
  style="width:100%; height:120px; border:0" ></iframe>

**Comment by Amosss** [comment]
From looking at the dates of the Open issues, I wonder if someone relevant is viewing this page...

**Comment by Amosss** [comment]
So I understand the developer doesn't really enter this site? For now sites are replaced by fireftp and I need to disable/enable (which means restart firefox) this addon every time....

**Comment by frog23** [comment]
> So I understand the developer doesn't really enter this site?

No, @mimecuvalo comes by every couple of weeks and then usually fixes several tickets at once. He is probably currently busy with other things, as this is not his full time job. So you have to be a bit patient.

By the way, is there a reason you embed the iframe using the FTP link and not the HTTP link, which seem to work just as well: http://tgftp.nws.noaa.gov/data/forecasts/taf/stations/EGBB.TXT vs [ftp://tgftp.nws.noaa.gov/data/forecasts/taf/stations/EGBB.TXT](ftp://tgftp.nws.noaa.gov/data/forecasts/taf/stations/EGBB.TXT). This looks like suitable workaround for me, providing that you control the content of the page or can contact the maintainer of the page and ask for the iframe URLs to be changed. 

But in general I agree with you, yes this is a bug and it should be fixed.

**Comment by leskaye** [comment]
Using http in the iframe does not work. I have tried it. Maybe there is a javascript work around to force the http get but that is beyond my pay grade!

**Comment by 85050** [comment]
Hi Amosss, this is really strange, I have the same problem also with http://www.mako.co.il/.  Did you figure out how to fix it?  I thought I did something wrong for this to happen.

**Comment by Amosss** [comment]
It's not strange, it's a bug :) I simply re-installed the previous version of fireftp, the problem is only with the latest one.

**Comment by 85050** [comment]
Hi, thanks for your response.  Which previous version of Fireftp did you use and where did you get it?

**Comment by 85050** [comment]
OK, I found ver. 2.0.26 and it works, thanks again!

**Comment by wrooot** [comment]
+1 this is a very annoying "feature". There should at least be an option to disable ftp link interception.

**Comment by Amosss** [comment]
It was reported 3 months ago, now we are just waiting for the developer to see this :) in the meantime, install the previous version, it works perfectly without this "feature"

**Comment by mimecuvalo** [comment]
Thanks for letting me know about the problem - I'll take a look into it. It seems it's because of some img links on the page that have 'ftp' in them but that shouldn't cause FireFTP load. Not sure why that would happen.

Development on FireFTP is slowing down since Firefox is dropping support for it in version 57 later in the year.  Hence, motivation for working on it isn't the highest currently. I'll probably be porting FireFTP at least to PaleMoon so that the extension survives.

Cheers!
Mime

**Comment by wrooot** [comment]
Firefox dropping this and that. I guess it's time to drop Firefox..

**Comment by Amosss** [comment]
What do you mean by "Firefox is dropping support for it in version 57"?
You are an addon' not part of FF itself

**Comment by mimecuvalo** [comment]
After 13 years and 25 million downloads later, Firefox has officially removed FireFTP and FireSSH support from the browser. Thus, I've ended support / development of the addons. I recommend switching to Waterfox to continue using the addons.

**Comment by frog23** [comment]
> Thus, I've ended support / development of the addons.
@mimecuvalo Sad to hear, but not unexpected. Thanks for all the work you did over the years and for making the world a better place, both by providing the software and by giving so much of the donations you received to charity. It was a fun ride, while it lasted.

**Comment by mimecuvalo** [comment]
<3 thanks for the kind words - it was indeed a fun ride!

**Comment by Amosss** [comment]
Count me in the so many thanks that you deserve for your work. The sad part is that there is no alternative to this addon
